backend/tests/v5/unit/schema/tickets/index.test.js

```diff
@@ -243,43 +244,64 @@ const testValidateTicket = () => {
 	describe('Validate ticket', () => {
 		const propertyTypeTestData = [
 			['Text', { type: propTypes.TEXT }, generateRandomString(), generateRandomString(121)],
+			['Text (unset)', { type: propTypes.TEXT }, null, generateRandomString(121), false],
```

Comment:
I think we should be splitting this in a different array and doing them separately

we want these to fail if it's new ticket, but pass when it's updating.

the good test bad test that is imposed this array also doesn't have much meaning.